### PR TITLE
Adds Base app with bottom navigation bar

### LINKED ISSUE #8: Base App
Es soll ein Grundgerüst für die App erstellt und designt werden, mit einer Navigation Bar und den Seiten “Home”, “Map” und “Tasks”. Es soll auch ein Button für Einstellungen geben.

### PR DESCRIPTION
**Pull request by AccsoMaximilianSaerov**

### CODE REVIEW

**Comment by remunds** [comment]
Userstory says that swiping is supposed to be possible.

File: lib/main.dart [hunk 1]
```diff
@@ -1,65 +1,103 @@
...
+                  MainAxisAlignment.end, //align the button to the right side
+              children: [
+                IconButton(
+                    //button initialisation
+                    icon: Icon(
+                      Icons.settings,
+                      color: Colors.black54,
+                    ),
+                    onPressed: () {
+                      //not yet implemented
```
Comment (remunds) [inline lib/main.dart]: add an opening window (user story)

**Review by remunds — CHANGES_REQUESTED** [review]

File: lib/main.dart [hunk 2]
```diff
@@ -1,66 +1,126 @@
...
-void main() {
-  runApp(App());
-}
+import 'Screens/settings.dart';
+
+void main() => runApp(MyApp()); //runs the main application widget
+
+/// This is the main application widget.
+class MyApp extends StatelessWidget {
+  static const String _title = 'Data Offloading App';
```
Comment (laura-jehn) [inline lib/main.dart:9]: should be title without underscore according to the dart style guide

File: lib/main.dart [hunk 3]
```diff
@@ -1,66 +1,126 @@
...
     );
   }
 }
 
-class MyHomePage extends StatefulWidget {
-  MyHomePage({Key key, this.title}) : super(key: key);
-  final String title;
+/// This is the stateful widget that the main application instantiates.
+class BaseAppWidget extends StatefulWidget {
+  BaseAppWidget({Key key}) : super(key: key); //muss ich noch herausfinden
```
Comment (laura-jehn) [inline lib/main.dart]: kein sinnvolles kommentar

**Review by laura-jehn — CHANGES_REQUESTED** [review]
- follows style guide
- sufficient commentary
- modular, bottom navigation bar in one file, page for 'settings' in separate file
- redundancy is okay
- code fulfills its purpose
- appropriate data structures
- good variable naming

**Review by laura-jehn — APPROVED** [review]

**Review by remunds — APPROVED** [review]

**Pull request merged**